registry/api/errcode/errors_test.go

```diff
@@ -109,28 +108,6 @@ func TestErrorsManagement(t *testing.T) {
 		t.Fatalf("unexpected json:\ngot:\n%q\n\nexpected:\n%q", string(p), expectedJSON)
 	}
 
-	// Now test the reverse
```

Comment:
I'm not sure that your personal beliefs matter here.

We have defined these errors and round trippable and these are part of the tests. It clearly looks to me like you added the `Args` field and deleted the failing tests.

This is not acceptable. I'm sorry.